f/connectors/alerts/tests/alerts_gcs_test.py

```diff
@@ -39,7 +40,7 @@ def _upload_blob(bucket, source_file_name, destination_blob_name):
         "100/raster/2023/09/S2_T1_202309900112345671.tif",
     ]
     for filename in alerts_filenames:
-        source_path = os.path.join(assets_directory, os.path.basename(filename))
+        source_path = Path(assets_directory) / Path(filename).name
```

Comment:
There is (and was before) a super subtle bug here, that would only raise itself if you're running the test suite on a Windows machine.  (At least I think this to be true, and I have seen something like it in the past before)

Basically the `/` operator will, on windows, become a `\`.  That's fine if you're constructing a path for the local filesystem, but this path is not a local file, it's a blob and needs to be `/`

One way (maybe there are others) to do this right would be

```suggestion
        source_path = posixpath.join(assets_directory, Path(filename).name)
```

with an `import posixpath` at top

Reply:
Not tested, but you might also have luck with:

```suggestion
        source_path = PosixPath(assets_directory) / PosixPath(filename).name
```

Reply:
I'll turn this into its own issue since we use this convention in many places in the repo.

Reply:
That's fine.  I think most of the places we use this convention, it is correct -- by which I mean we are referring to the local file system, or a network mount that's exposed as a local file system.

This is one of the few (only) cases I'm aware of where we're dealing with blob storage names.